src/BottomNavigation/BottomNavigation.js

```diff
@@ -0,0 +1,54 @@
+// @flow weak
+
+import React, { PropTypes, Children, cloneElement } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+
+export const styleSheet = createStyleSheet('BottomNavigation', (theme) => {
+  return {
+    root: {
+      display: 'flex',
+      justifyContent: 'center',
+      height: 56,
+      backgroundColor: theme.palette.background.paper,
+    },
+  };
+}, { index: 5 });
```

Comment:
The index defines the CSS injection order. It's something critical that you don't want to get wrong.
@nathanmarks Know it better than me, but it seems like we are only using negative values.
`-10` sounds like a good value. But I'm not sure about it.